examples/cd_distillation_cifar10.ipynb

```diff
@@ -0,0 +1,781 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plot_labels = ['0', '1', '2', '3', '4', '5']\n",
+    "\n",
+    "N = 6\n",
+    "ind = np.arange(N)\n",
+    "width = .35\n",
+    "\n",
+    "fig_bar_cd, ax = plt.subplots()\n",
+    "ax2 = ax.twinx()\n",
+    "\n",
+    "p0 = ax.bar(ind[0], score_x.mean(), yerr=score_x.std(), capsize=2)\n",
+    "p1 = ax.bar(ind[1:], mu_noharm, width, yerr=std_noharm, capsize=2)\n",
+    "p2 = ax.bar(ind[1:] + width, mu_harm, width, yerr=std_harm, capsize=2)\n",
+    "\n",
+    "ax.set_title('Harmfullness Scores and Accuracy by Corruption Severity')\n",
+    "ax.set_xticks(ind + width / 2)\n",
+    "ax.set_xticklabels(plot_labels)\n",
+    "ax.set_ylim((-2))\n",
+    "ax.legend((p1[0], p2[0]), ('Not Harmful', 'Harmful'), loc='upper right', ncol=2)\n",
+    "ax.set_ylabel('Score')\n",
+    "ax.set_xlabel('Corruption Severity')\n",
+    "\n",
+    "color = 'tab:red'\n",
+    "ax2.set_ylabel('Accuracy', color=color)\n",
+    "ax2.plot(acc, color=color)\n",
+    "ax2.tick_params(axis='y', labelcolor=color)\n",
+    "\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Plot p-values for contaminated batches\n",
+    "\n",
+    "In order to simulate a realistic scenario, we perform a K-S test on batches of instance which are increasingly contaminated with corrupted data. The following steps are implemented:\n",
+    "\n",
+    "* We randomly pick `n_ref` samples from the non-currupted test set to be used as a reference set in the initialization of the K-S drift detector.  \n",
+    "\n",
+    "* We sample batches of data of size `batch_size` contaminated with an increasing number of harmful corrupted data and harmless corrupted data. \n",
+    "\n",
+    "* We apply the K-S test to these batches and obtain the p-values for each of them. \n",
+    "\n",
+    "* We observe that contamination of the batches with harmful data reduces the p-values much faster than contamination with harmless data, in which cases the p-values remains above the detection treshold even when the batch is heavily contaminated.  \n",
+    "\n",
+    "In this examples, each batch containes 100 instances where a fraction p consists of harmful or harmless corrupted data. The instances are sampled randomly from the test set and the harmfull or harmless corrupted set. For each batch a K-S test is performed with respect to a reference set of 1000 instance sampled from the uncorrupted test set. The K-S test returns a p-values which represents the probability that the batch and the reference set are sampled from the same probability distribution. \n",
```

Comment:
"The K-S test returns a p-values which represents the probability that the batch and the reference set are sampled from the same probability distribution."
This is not exactly right. p-values are the probability of obtaining results at least as extreme as the observed results under the assumption that the null hypothesis holds.